velox/exec/AggregationHook.h

```diff
@@ -16,6 +16,7 @@
 #pragma once
 
 #include "velox/common/base/Range.h"
+#include "velox/functions/prestosql/CheckedArithmeticImpl.h"
```

Comment:
@majetideepak I think this introduces a circular dependency between exec and prestosql. It is quite unfortunate that hooks are tightly coupled with execution and are not part of the aggregate function's implementation. A simple way out could be to copy-paste checkedPlus logic.

CC: @oerling

Reply:
I moved the `CheckArithmetic.h` to `common/base` in the newest master